reference/5.1/Microsoft.PowerShell.Core/About/about_Scopes.md

```diff
@@ -191,10 +191,41 @@ Using is a special scope modifier that identifies a local variable in a remote
 command. Without a modifier, PowerShell expects variables in remote commands
 to be defined in the remote session.
 
-The Using scope modifier is introduced in PowerShell 3.0.
+The `Using` scope modifier is introduced in PowerShell 3.0.
+
+r any script or command that executes out of session, you need the `Using`
```

Comment:
Looks like "Fo" is missing from the beginning of this sentence.

Reply:
Fixed